qp_qiime2/__init__.py

```diff
@@ -34,3 +35,25 @@
     rarefy, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
+
+# Define the beta_diversity command
+req_params = {'i-table': ('artifact', ['BIOM'])}
+opt_params = {
+    'p-metric': [
+        ('choice:["sokalsneath", "kulsinski", "mahalanobis", "matching", '
+         '"euclidean", "correlation", "yule", "russellrao", "hamming", '
+         '"jaccard", "braycurtis", "dice", "rogerstanimoto", "sqeuclidean", '
+         '"cityblock", "sokalmichener", "cosine", "wminkowski", "seuclidean", '
+         '"chebyshev", "canberra", "unweighted", "weighted-normalized", '
+         '"weighted-unnormalized"]'), 'jaccard'],
+    'i-tree': ['choice:["default"]', 'default']}
```

Comment:
Since the default metric is not phylogenetic, should the default here be "No tree" or something along those lines?